conf/cassandra.yaml

```diff
@@ -1343,3 +1343,64 @@ enable_sasi_indexes: false
 # Enables creation of transiently replicated keyspaces on this node.
 # Transient replication is experimental and is not recommended for production use.
 enable_transient_replication: false
+
+# Emulates database-as-a-service defaults.
+#
+# When enabled, some guardrails defaults are modified to values that are appropriate for cloud environments.
+# This includes (but is not limited to) stricter guardrails defaults.
+#
+# This can be used as an convenience to develop and test applications meant to run in a cloud environment.
+# emulate_dbaas_defaults: false
```

Comment:
The name `emulate_dbass_defaults` is confusing IMO
It doesn't emulate things, it _applies different defaults_

Reply:
renamed to `apply_dbaas_defaults`